presto-docs/src/main/sphinx/release/release-0.205.rst

```diff
@@ -0,0 +1,72 @@
+=============
+Release 0.205
+=============
+
+General Changes
+---------------
+
+* Fix parsing of row type signatures with spaces and types containing spaces.
```

Comment:
/cc @caithagoras

Reply:
This allow users to create columns/expressions with types such as `row(timestamp with time zone)`. For example, allow user to do casts:
```
cast(row(timestamp '2018-06-01') as row(timestamp with time zone))
```.
Before the fix, this cast will fail since `timestamp` is treated as field name and type `with time zone` does not exists